lib/studies/reply/mail_card_preview.dart

```diff
@@ -0,0 +1,113 @@
+import 'package:flutter/material.dart';
+import 'package:animations/animations.dart';
+import 'package:gallery/layout/adaptive.dart';
+import 'package:gallery/studies/reply/colors.dart';
+import 'package:gallery/studies/reply/mail_view_page.dart';
+
+const _assetsPackage = 'flutter_gallery_assets';
+const _iconAssetLocation = 'reply/icons';
+
+class MailCardPreview extends StatelessWidget {
+  const MailCardPreview({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final isDesktop = isDisplayDesktop(context);
+    final textTheme = Theme.of(context).textTheme;
+
+    return OpenContainer(
+      openBuilder: (context, closedContainer) {
+        return const MailViewPage();
+      },
+      closedShape: const RoundedRectangleBorder(
+        borderRadius: BorderRadius.all(Radius.circular(0)),
+      ),
+      closedElevation: 0,
+      closedBuilder: (context, openContainer) {
+        return InkWell(
+          onTap: openContainer,
+          child: Row(
+            children: [
+              Expanded(
+                child: Padding(
+                  padding: const EdgeInsets.all(20),
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                    children: [
+                      Row(
+                        children: [
+                          Text('Google Express', style: textTheme.caption),
+                          Padding(
+                            padding: const EdgeInsetsDirectional.only(
```

Comment:
This could just be `const EdgeInsets.symmetric(horizontal: 4)` to avoid duplicating the 4